NSelene/SeleneCollection.cs

```diff
@@ -60,6 +60,7 @@ public SeleneCollection Should(Condition<SeleneCollection> condition)
             return Selene.WaitFor(this, condition);
         }
 
+        [Obsolete("Use the negative condition instead")]
```

Comment:
let's provide an example in such messages, like:
[Obsolete("Use the negative condition instead, e.g. Should(Be.Not.Visible) instead ShouldNot(Be.Visible)")]